### ISSUE TITLE
Redis layer missing on PHP 8.1

### ISSUE THREAD

**Issue by aran112000** [brefphp/extra-php-extensions#281]
The Redis layer which was added by @shouze under the following commit for PHP 8.1 doesn't appear to actually work:
https://github.com/brefphp/extra-php-extensions/commit/f5fe4498b58d662c2725896373ea5ee6de015564

When registered for a function PHP reports the Redis function never loaded - verified with `get_loaded_extensions()`. No sign of any startup errors from PHP which I would have expected, any idea how to debug/resolve this?

**Comment by sbennett-ihasco** [comment]
I noticed the same thing building a redis+igbinary extension for 8.1. Staying with 8.0 for now.

**Comment by aran112000** [comment]
@sbennett-ihasco we had other issues with 8.0 as the Memcached extension is missing - I've seen there's a MR sorting that out awaiting the next release

**Comment by aran112000** [comment]
Managed to get the startup errors outputting which confirms both Redis and Memcached were compiled against the wrong PHP version:
```
PHP Warning:  PHP Startup: memcached: Unable to initialize module
Module compiled with module API=20201009
PHP    compiled with module API=20210902

These options need to match in Unknown on line 0
```
```
PHP Warning:  PHP Startup: redis: Unable to initialize module
Module compiled with module API=20201009
PHP    compiled with module API=20210902

These options need to match in Unknown on line 0
```

**Comment by aran112000** [comment]
closed by #282
